client/my-sites/stats/activity-log/index.jsx

```diff
@@ -100,6 +100,13 @@ class ActivityLog extends Component {
 		window.scrollTo( 0, 0 );
 	}
 
+	handlePeriodChange = ( { date, direction } ) => {
+		this.props.recordTracksEvent( 'calypso_activitylog_monthpicker_change', {
+			date,
```

Comment:
I take that back 🤦‍♂️ I just saw it. Thanks for catching this, I'll track it down.